javascript/atoms/dom.js

```diff
@@ -567,43 +567,34 @@ bot.dom.isShown_ = function(elem, ignoreOpacity, parentsDisplayedFn) {
 bot.dom.isShown = function(elem, opt_ignoreOpacity) {
   var displayed;
 
-  if (bot.dom.IS_SHADOW_DOM_ENABLED) {
-    // Any element with a display style equal to 'none' or that has an ancestor
-    // with display style equal to 'none' is not shown.
-    displayed = function(e) {
-      if (bot.dom.getEffectiveStyle(e, 'display') == 'none') {
-        return false;
-      }
-      var parent;
-      do {
-        parent = bot.dom.getParentNodeInComposedDom(e);
-        if (parent instanceof ShadowRoot) {
-          if (parent.host.shadowRoot != parent) {
-            // There is a younger shadow root, which will take precedence over
-            // the shadow this element is in, thus this element won't be
-            // displayed.
-            return false;
-          } else {
-            parent = parent.host;
-          }
-        } else if (parent && (parent.nodeType == goog.dom.NodeType.DOCUMENT ||
-            parent.nodeType == goog.dom.NodeType.DOCUMENT_FRAGMENT)) {
-          parent = null;
-        }
-      } while (elem && elem.nodeType != goog.dom.NodeType.ELEMENT);
-      return !parent || displayed(parent);
-    };
-  } else {
-    // Any element with a display style equal to 'none' or that has an ancestor
-    // with display style equal to 'none' is not shown.
-    displayed =  function(e) {
-      if (bot.dom.getEffectiveStyle(e, 'display') == 'none') {
+  // Any element with a display style equal to 'none' or that has an ancestor
+  // with display style equal to 'none' is not shown.
+  displayed = function(e) {
```

Comment:
nit: remove the `var displayed;` declaration above and change this to:

```js
/**
 * @param {?Node} e
 * @return {boolean}
 */
function displayed(e) {
```

The extra type annotations are annoying, but helps improve the compiler's accuracy in type checking